apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -0,0 +1,399 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { z } from 'zod';
+import { assetPath, compiledScriptTag, nodeModulesAssetPath } from '../../lib/assets';
+import { AssessmentQuestionSchema, IdSchema, TopicSchema } from '../../lib/db-types';
+
+export const AssessmentQuestionRowSchema = AssessmentQuestionSchema.extend({
+  alternative_group_number_choose: z.number().nullable(),
+  alternative_group_number: z.number().nullable(),
+  alternative_group_size: z.number(),
+  assessment_question_advance_score_perc: z.number().nullable(),
+  avg_question_score_perc: z.number().nullable(),
+  display_name: z.string().nullable(),
```

Comment:
When will this be null?

Reply:
I answered my own question 😅 https://github.com/PrairieLearn/PrairieLearn/pull/9681